sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -58,24 +58,24 @@ az cognitiveservices account create \
     --location westus2 \
     --yes
 ```
-### Authenticate the client
-In order to interact with the Text Analytics service, you will need to create an instance of the `TextAnalyticsClient` 
+In order to interact with the Text Analytics service, you will need to create an instance of the [TextAnalyticsClient][text_analytics_sync_client] 
 class. You will need an **endpoint** and either an **API key** or **AAD TokenCredential** to instantiate a client 
 object. And they can be found in the [Azure Portal][azure_portal] under the "Quickstart" in your created
 Text Analytics resource. See the full details regarding [authentication][authentication] of Cognitive Services.
 
 #### Get credentials
 The authentication credential may be provided as the API key to your resource or as a token from Azure Active Directory.
 
-
-##### **Option 1**: Create TextAnalyticsClient with AzureKeyCredential
-To use AzureKeyCredential authentication, provide the [key][key] as a string to the [AzureKeyCredential][azure_key_credential]. This can be found in the [Azure Portal][azure_portal] 
-   under the "Quickstart" section or by running the following Azure CLI command:
+##### **Option 1**: Create TextAnalyticsClient with API Key Credential
```

Comment:
I think you should add a section about how to get the key, like .net [has](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/README.md#get-api-key)

Reply:
With newest update, it is included in "Create a Text Analytics client with key credential". And we should no duplicate it in two sections with same content. it might confuse user.